source/tests/llassetgen-tests/Image.cpp

```diff
@@ -0,0 +1,129 @@
+#include <ft2build.h>  // NOLINT include order required by freetype
+#include FT_FREETYPE_H
+
+#include <gmock/gmock.h>
+#include <llassetgen/llassetgen.h>
+
+#include <fstream>
+
+using namespace llassetgen;
+
+// current working directory: openll-asset-generator/build/source/tests
+std::string test_source_path = "../../../source/tests/llassetgen-tests/testfiles/";
+std::string test_destination_path = "../../";
```

Comment:
This relies on the location of the cmake build dir compared to the source directory, making it hard to run tests while developing (especially with an IDE). Could we maybe pass the path to the source directory in using CMake (cmake variable -> c++ define -> here)?

In a perfect world we would also use the temp dir as the destination, or just clean up the files after the tests.